### ISSUE TITLE
can‘T

### ISSUE THREAD

**Issue by Bitllion** [RadeonOpenCompute/ROCm#889]

**Comment by Bitllion** [comment]
/opt/rocm/bin/rocminfo
/opt/rocm/opencl/bin/x86_64/clinfo

ROCk module is loaded
root is member of video group
hsa api call failure at: /data/jenkins_workspace/compute-rocm-rel-2.7/rocminfo/rocminfo.cc:1102
Call returned HSA_STATUS_ERROR_OUT_OF_RESOURCES: The runtime failed to allocate the necessary resources. This error may also occur when the core runtime library needs to spawn threads or create internal OS-specific events.
Number of platforms:                             1
  Platform Profile:                              FULL_PROFILE
  Platform Version:                              OpenCL 2.1 AMD-APP (2949.0)
  Platform Name:                                 AMD Accelerated Parallel Processing
  Platform Vendor:                               Advanced Micro Devices, Inc.
  Platform Extensions:                           cl_khr_icd cl_amd_event_callback cl_amd_offline_devices


  Platform Name:                                 AMD Accelerated Parallel Processing
ERROR: clGetDeviceIDs(-1)

**Comment by ITfirewall** [comment]
Hey Bitllion, 

 Whats your OS and Gpu???

**Comment by jppyykm** [comment]
I have a similar problem. I am running Ububtu and my gpu is RX 480. 
```
hsa api call failure at: /data/jenkins_workspace/compute-rocm-rel-2.7/rocminfo/rocminfo.cc:1102
Call returned HSA_STATUS_ERROR_OUT_OF_RESOURCES: The runtime failed to allocate the necessary resources. This error may also occur when the core runtime library needs to spawn threads or create internal OS-specific events.
```
https://rocm.github.io/ROCmInstall.html  <-- Following installation 
 
Installing this thing has been really, really annoying. I broke my Mint installation a few times tinkering to make it run there. I got tired and installed Ubuntu so it is supported. Now this.

**Comment by ITfirewall** [comment]
Hi jppyykm,

Can you try again with an older version of ROCm driver??? ( try ROCm 2.6, 2.5,....) from repository.
I have a different problem with ubuntu 16.04 + ROCm 2.7.1, this is causing a problem with OpenCL.

**Comment by jppyykm** [comment]
Hi! I am having all sorts of trouble. I am working on it in sprints.. getting this thing to work is urgh,,

**Comment by b4551k5** [comment]
I got the same issue. It says:

"hsa api call failure at: /data/jenkins_workspace/compute-rocm-rel-2.9/rocminfo/rocminfo.cc:1102
Call returned HSA_STATUS_ERROR_OUT_OF_RESOURCES: The runtime failed to allocate the necessary resources. This error may also occur when the core runtime library needs to spawn threads or create internal OS-specific events."

I installed rocm-dev since my kernel seems to be not supported by rocm-dkms anyway.
